eox_core/api/v1/urls.py

```diff
@@ -1,11 +1,9 @@
 """ urls.py """
 
 from django.conf.urls import url
-from rest_framework_jwt.views import obtain_jwt_token
 from eox_core.api.v1 import views
 
 urlpatterns = [  # pylint: disable=invalid-name
-    url(r'^api-token-auth/', obtain_jwt_token),
     url(r'^user/$', views.EdxappUser.as_view(), name='edxapp-user'),
     url(r'^enrollment/$', views.EdxappEnrollment.as_view()),
     url(r'^userinfo$', views.UserInfo.as_view()),
```

Comment:
i think it should be set the name value to each url definition, just for a better reverse operation.

Reply:
Fixed in latest commit.